docs/immortal-empires/beastmen.md

```diff
@@ -0,0 +1,58 @@
+# Beastmen
+
+## Taurox the Brass Bull: Slaughterhorn Tribe
+
+**Ideas**
+
+  * Objectives
+    * There are champions in the cold North who claim the strength of their arm is unmatched. Show them what true strength is.
+      * Destroy factions: Naggarond, Legion of the Gorequeen
+
+    * The Chaos Gods will see his foes smote upon ruined lands, and they will see in him greater ruination yet. His fury is endless, but the world is not.
+      * Accumulate N Marks of Ruination
+    * The world will know of the Brass Bull. His name will inspire dread in all those who hear it, no matter how far they are from his Bloodgrounds. 
+      * Accumulate N Dread
+
+    * An Ocean separates Men from your doom, but it is not enough to save them. Cross the great ocean, tear down their illusion of safety.
+      * Sack or Raze Altdorf, Marienburg, etc
+
+  * Rewards
+    * Great Beasts - Taurox and his beasts grew mighty indeed, tearing across vast lands and razing all in their way. The weaker beasts quickly failed out of the horde, forgotten or dead, leaving only the strong.
+      * Bonuses for Minotaurs (all)
+      * Minor bonuses for Cygor/Ghorgon
+        * Unbreakable? "Veterans of many battles, unshakeable and immovable"
+        * MD
+
+    * Greater Monsters - Even beasts of fury and strength can't hope to be favoured over chaos-tainted creatures of great size and power.
+      great and foul monstrosities.
+      * Bonuses for Razorgore/Harpies/Warhands/Manticore/Spawn/Jabberslythe
+        * MA/MD
+
+**Route I - title1*
+  * objective1
+  * objective2
+  * objectiveN
+  * **Reward:** 
+    * reward1: 
+    * rewardN:
+
+**Route II - title2*
+  * objective1
+  * objective2
+  * objectiveN
+  * **Reward:** 
+    * reward1: 
+    * rewardN:
+
+**Route III - title3*
+  * objective1
+  * objective2
+  * objectiveN
+  * **Reward:** 
+    * reward1: 
+    * rewardN:
+
+Reward idea: spread khorne corruption
+Taurox now counts as a Khorne unit for the purposes of Khorne corruption, which means that he’ll be even stronger in certain regions. Khorne corruption empowers all Khornate forces with more melee attack, weapon strength, and charge bonus. It afflicts non-Khorne factions with less melee defence.
```

Comment:
So this would only affect Taurox, and not his units/armies/lords/heroes? Are they immune to high Khorne corruption?

Reply:
No no no. This was me quoting a patch note of immortal empires. This is not the vco mod.

I was trying to explain why we want to ally with Khorne not attack them. Because of this change they made. Players will not want to remove sources of khorne corruption as taurox.